index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HANGMAN</title>
+    <meta name="description" content="hangman">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Coiny&amp;subset=latin-ext" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class = 'container'>
+      <div class = 'header'>
+        <h1 class = 'title'>HANGMAN</h1>
+        <button class = 'btn'>START GAME</button>
+      </div>
+      <div class = 'game-wrapper'>
+        <div class = 'image-container'>
```

Comment:
there could be improcements for naming. I recommend to use everywhere `wrapper` instead of `container`.